lib/orangelight/browse_lists.rb

```diff
@@ -91,7 +91,11 @@ def browse_cn(_sql_command, facet_request, conn, facet_field, _table_name)
                 date = record['pub_created_display'][0]
               end
               label = cn
-              author = record['author_display'][0..1].last if record['author_display']
+              if record['author_display']
+                author = record['author_display'][0]
```

Comment:
Can you keep the line as `record['author_display'][0..1].last`? This will display the vernacular script over the Latin script when it is present in the record.

Reply:
yes I will. thanks.